addons/docs/package.json

```diff
@@ -78,10 +78,16 @@
     "ts-dedent": "^1.1.1",
     "util-deprecate": "^1.0.2",
     "vue-docgen-api": "^4.7.0",
-    "vue-docgen-loader": "^1.4.0"
+    "vue-docgen-loader": "^1.4.0",
+    "@angular/common": "8.2.14",
+    "@angular/compiler": "8.2.14",
+    "@angular/platform-browser": "8.2.14",
+    "@angular/platform-browser-dynamic": "8.2.14",
+    "reflect-metadata": "0.1.13",
+    "zone.js": "^0.10.2",
+    "@angular/core": "^9.1.0"
```

Comment:
Consider using the same version for all `@angular`-namespaced packages. I noticed @angular/core is 9, and the rest are 8

Reply:
Good call, will do soon!